### ISSUE TITLE
Error while updating propety underlineColorAndroid of a view managed by AndroidTextInput

### ISSUE THREAD

**Issue by RobertArissen** [reactseals/react-native-leanback#17]
Hello,

If i use the style and value setup you have added in you'r project the app will crash if we use a Textinput. This also happend by the example project.

What do i wrong?

**Comment by sharan-SKY** [comment]
Did you manage to resolve this error?

**Comment by RobertArissen** [comment]
No, i stop using this package because of this problem. The rest of the package works perfect for me :(
